content/help/ops/security/authorization-permissive/index.md

```diff
@@ -4,14 +4,17 @@ description: Authorization is enabled, but requests make it through anyway.
 weight: 50
 ---
 If authorization checks are enabled for a service and yet requests to the service aren't being blocked, then
-its likely that authorization was not actually enabled successfully. You can do the following to verify:
+authorization was likely not enabled successfully. To verify, follow these steps:
 
-1. Check [this page](/docs/concepts/security/#authorization) to find out how to correctly enable Istio authorization policy. Use `istioctl` instead of `kubectl`, because `istioctl` enforces an additional validation check to prevent errors caused by typos.
+1. Check [enable authorization docs](/docs/concepts/security/#enabling-authorization) to correctly enable
+Istio authorization.
 
-1. Avoid enabling authorization for Istio Control Planes Components, including Mixer, Pilot, Ingress. Istio authorization policy is designed for authorizing access to services in Istio Mesh. Enabling it for Istio Control Planes Components may cause unexpected behavior.
+1. Avoid enabling authorization for Istio Control Planes Components, including Mixer, Pilot, Ingress.
+Istio authorization is designed for authorizing access to services in Istio Mesh. Enabling it
```

Comment:
Articles and pronoun issues:
The Istio authorization features are designed for authorizing access to services in an Istio Mesh. Enabling the authorization features